components/cli/pkg/commands/push.go

```diff
@@ -123,6 +141,42 @@ func RunPush(cellImage string, username string, password string) {
 	util.PrintWhatsNextMessage("pull the image", "cellery pull "+cellImage)
 }
 
+func pushDockerImages(dockerImages []string, username string, password string) {
+	for _, elem := range dockerImages {
+		spinner := util.StartNewSpinner("Pushing Docker image " + util.Bold(elem))
+		cmd := exec.Command("docker", "push", elem)
```

Comment:
Shouldn't we give the username and password as flags here ? If the user had not logged in this would fail.

Reply:
As per the current implementation, the user has to do a docker login to push the images.